src/coreclr/vm/loongarch64/thunktemplates.S

```diff
@@ -4,20 +4,18 @@
 #include "unixasmmacros.inc"
 #include "asmconstants.h"
 
-; Note that the offsets specified in pcaddi must match the behavior of GetStubCodePageSize() on this architecture/os. 
```

Comment:
Can you please keep this comment? It is still correct. If we ever decided to change the `GetStubCodePageSize()` return value to something larger than 16kB, this would need to be updated.

Reply:
Ok, thanks
I will update it.